developer_setup.md

```diff
@@ -37,8 +37,17 @@
   sudo passwd postgres
   # ↑ will ask to choose a password
   sudo postgresql-setup initdb
+  # if the above usage of postgresql-setup fails with 'command not found', use the following command with the relevant postgres 9.x version - below an example for postgres 9.1:
+  # for rhel:
+  sudo service postgresql-9.1 initdb
+  # for fedora:
+  sudo systemctl postgresql-9.1 initdb
```

Comment:
Either way `systemctl <service> <action>` is not proper syntax for the command and will always fail.